### DISCUSSION TITLE
Pass a variable to a partial

### DISCUSSION THREAD

**Discussion by stephenmeehanuk** [statamic/cms#5994] [General]
I'm using a global to control a 404 page. In the global, there's a handle called `{{ button_text }}`. I can output this by using `{{ 404:button_text }}`, but I can't seem to use it as a variable in a partial?

This works, but, I'd like to control `button_text` via the CMS
```
 {{ partial src="_components/button/button" button_text="Return to homepage"  }}
 ```
This doesn't work
```
 {{ partial src="_components/button/button" button_text="{ 404:button_text }"  }}
 ```

`laravel.log` shows this message
```
[2022-05-04 18:44:06] local.DEBUG: Cannot render an array variable as a string: {{ button_text }}  
```

Is there a way to make this work?

 
 ```
Statamic 3.3.10 Pro
Laravel 9.5.1
PHP 8.0.8
simonhamp/statamic-heroicons 1.0.1
statamic/collaboration 0.4.0

Parser: runtime
```

**Comment by jackmcdade** [comment]
Try `{{ partial src="_components/button/button" :button_text="404:button_text"  }}`.

**Comment by JohnathonKoster** [comment]
Replicated, and is bugged (only happens when param variables start with numbers or reach into the Cascade. This does both so was a fantastic find!). Corrected with this PR: https://github.com/statamic/cms/pull/5995

**Reply by stephenmeehanuk** [reply]
Great stuff!, Thanks for taking a look.
